site/content/docs/howto/install-concierge.md

```diff
@@ -12,29 +12,37 @@ menu:
 This guide shows you how to install the Pinniped Concierge.
 You should have a [supported Kubernetes cluster]({{< ref "../reference/supported-clusters" >}}).
 
+In the examples below, you can replace *{{< latestversion >}}* with your preferred version number.
+You can find a list of Pinniped releases [on GitHub](https://github.com/vmware-tanzu/pinniped/releases).
+
 ## With default options
 
-1. Install the latest version of the Concierge into the `pinniped-concierge` namespace with default options:
+**Warning:** the default Concierge configuration may create a public LoadBalancer Service on your cluster if that is the default on your cloud provider.
```

Comment:
Nice warning. Should we say which cluster types this happens on? And maybe point them towards the docs for the configuration API that can be used for controlling this?

Reply:
The only docs we have for that are inline in the `values.yaml` file, so I think we are essentially linking to the right place already:
https://github.com/vmware-tanzu/pinniped/blob/main/deploy/concierge/values.yaml#L67

I added a sentence giving some examples of what options are in `values.yaml` to draw a bit more attention to it.